ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -1285,17 +1353,48 @@ function ListRules(props) {
   }, [hashQuery, filteredRulesCount]);
 
   return (
-    <Dashboard title="Rules">
+    <Dashboard
+      title={
+        rewindDate ? `Rules @ ${rewindDate.toString().split('(')[0]}` : 'Rules'
+      }>
       {isLoading && <Spinner loading />}
       {error && <ErrorPanel fixed error={error} />}
       {!isLoading && productChannelOptions && (
         <Fragment>
           <div ref={searchFieldRef} className={classes.options}>
+            <DateTimePicker
+              className={classes.rewindPicker}
+              disableFuture
+              inputVariant="outlined"
+              label="Rewind to..."
```

Comment:
What about "Rule database state as of YYYY/DD/MM HH:MM AM/PM"?

Reply:
I think rewind to might be more clear for the label when there is no rewind date selected.